deps/sudo_install_tables.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
One more correction. Can we add these lines before any installation lines to prevent unnecessary sudo calls if tables lib is already installed?
```
python_tables_installed=`pip list | grep '^tables (' | wc -l`
if [ "$python_tables_installed" > "0" ]; then
    exit 0
fi
echo "Installing python tables"
```